src/Security/Authentication/JwtBearer/src/JwtBearerConfigureOptions.cs

```diff
@@ -54,10 +54,11 @@ public void Configure(string? name, JwtBearerOptions options)
 
     private static SecurityKey GetIssuerSigningKey(IConfiguration configuration, string? issuer)
     {
-        var jwtKeyMaterialSecret = configuration[$"{issuer}:KeyMaterial"];
+        var jwtKeyMaterialSecret = configuration[$"{issuer}:SigningKey:Value"];
```

Comment:
I don't think `issuer` should still be here. Are we missing some test coverage for this code?